### ISSUE TITLE
aria announcement throw error when the grid destroyed

### ISSUE THREAD

**Issue by Ksisu** [ag-grid/ag-grid#7647]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**

I use ag-grid inside a MUI dialog. After upgrade from 31.0.3 to 31.1.0 when i close the dialog, i got an error `TypeError: Cannot set properties of null (setting 'textContent')` from here (inside the setTimeout at line 39):
https://github.com/ag-grid/ag-grid/blob/9e87c3e12ca0a70db5985efd37eed8c07e46d686/grid-community-modules/core/src/ts/rendering/ariaAnnouncementService.ts#L33-L41

I think it is related to [AG-10049](https://github.com/ag-grid/ag-grid/commit/120be07f6e33cb9150ba882d653d3bd49ed0bbe1) when the timeout was increased from 5 to 50. With 5 the code ran before the dialog close animation finished, but with 50 the gird is destroyed when the code run.

I think it just needs `if (!this.descriptionContainer) { return; }` check inside the setTimeout functions too.

I am happy to create a PR for this if you agree with my suggestion. 

**Expected behavior**
Should not throw error.

**Please tell us about your environment:**

* **AG Grid version:** 31.1.0

* **Browser:**
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36'

* **Language:** all

**Comment by AG-Zoheil** [comment]
Hello @Ksisu.

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference: **AG-10828**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil

**Comment by aravindshetty21** [comment]
I encountered the same issue today. Any update on this ?

**Comment by AG-Zoheil** [comment]
Hi @aravindshetty21,

**AG-10828** was closed in our release 31.2. If you are still having issues on versions after this please provide a plunker that replicates the problem and we can have a look.

Kind regards,
Zoheil
